packages/toolkit/form-field/form-field-wrapper.css

```
@@ -417,35 +442,21 @@
   --_outline-min-height: var(
-    --ngx-form-field-outline-min-height,
-    var(
-      --ngx-form-field-min-height,
-      calc(
-        var(--_label-outline-line-height) + var(--_label-outline-gap) +
-          var(--_input-outline-line-height) + var(--_padding-vertical) +
-          var(--_padding-vertical)
-      )
+    --ngx-form-field-min-height,
+    calc(
+      var(--_label-outline-line-height) + var(--_label-outline-gap) +
```

Stylelint (17.7.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)